pages/common/ghci.md

```diff
@@ -0,0 +1,24 @@
+# ghci
+
+> The Glasgow Haskell Compiler's interactive environment.
+> More information: <https://downloads.haskell.org/ghc/latest/docs/html/users_guide/ghci.html>.
+
+- Start an interactive REPL session:
+
+`ghci`
+
+- Start an interactive REPL session and load the specified Haskell source file:
+
+`ghci {{source_file.hs}}`
+
+- Start an interactive REPL session and enable a language option:
+
+`ghci -X{{language_option}}`
+
+- Start an interactive REPL session and enable almost all compiler warnings:
+
+`ghci -Wall`
```

Comment:
I can't verify this argument, even with the command installed. The official man page is not friendly to me.

Reply:
I verified it by starting `ghci -Wall` and evaluating the expression `2*2`. This immediately triggers a warning which would not be there without `-Wall`.

Reply:
@gutjuri Oh cool, and How can I test the `-X` example then?

Reply:
For example, start `ghci -XNoImplicitPrelude` and try calling something from the Prelude (e.g. `take 5 [1..]`). 

This should also work with any other language option, though many of them are not easily testable in a REPL in my opinion.

Reply:
Oh `all` is an argument option. Based on [sonar-scanner](https://github.com/tldr-pages/tldr/blob/master/pages/common/sonar-scanner.md):
```suggestion
`ghci -W{{all}}`
```